aikau/src/test/resources/alfresco/lists/AlfHashListTest.js

```diff
@@ -195,6 +196,37 @@ define(["alfresco/TestCommon",
 
          "Post Coverage Results": function() {
             TestCommon.alfPostCoverageResults(this, browser);
+         },
+
+         "Payload Data Loading is prevented when the hash vars are filter out": function() {
+            var badUrl = TestCommon.testWebScriptURL("/AlfHashListPayloadData#lib=Libraries&path=a/b/c"),
+                goodUrl = TestCommon.testWebScriptURL("/AlfHashListPayloadData#lib=Personal&path=d/e/f");
+
+            return browser.findByCssSelector("body")
+               .clearLog()
+               .end()
+
+            .get(badUrl)
+               .findByCssSelector("body")
+               .end()
+
+            .getLastPublish("ALF_RETRIEVE_DOCUMENTS_REQUEST")
+               .then(function(payload) {
+                  assert.notProperty(payload, "path", "There should be no path value in the payload");
+               })
+
+            .findByCssSelector("body")
+               .clearLog()
+               .end()
+
+            .get(goodUrl)
```

Comment:
One minor change... could you break this test up... and add "Post Coverage Results (1)" before reloading the page with the good URL....